packages/components/psammead-brand/src/index.stories.jsx

```diff
@@ -15,8 +15,23 @@ storiesOf('Brand', module)
         .filter(key => key !== 'BBC_BLOCKS')
         .map(key => key.charAt(0).toUpperCase() + key.slice(1));
 
-      const choice = select('Service SVG', options, 'news').toLowerCase();
-      return <Brand brandName="Default Brand Name" svg={svgs[choice]} />;
+      const choice = select('Service SVG', options, 'News').toLowerCase();
+      const svgRatio = svgs[choice].ratio;
+      const svgMaxHeight = 24;
+      const svgMinHeight = 16;
+      const minWidth = number('minimum svg width', svgRatio * svgMinHeight);
+      const maxWidth = number('maximum svg width', svgRatio * svgMaxHeight);
+      const height = number('desired height svg', svgMaxHeight);
+
+      return (
+        <Brand
+          brandName="Default Brand Name"
+          height={height}
+          minWidth={minWidth}
+          maxWidth={maxWidth}
+          svg={svgs[choice]}
+        />
+      );
     },
     { notes },
   )
```

Comment:
The `without brand svg` story errors since required props are not passed in:
<img width="575" alt="Screen Shot 2019-05-03 at 17 58 06" src="https://user-images.githubusercontent.com/3028997/57153027-1fc15d00-6dcd-11e9-99f9-ad8f2473ecf1.png">

Reply:
I dont think we need a story for a missing svg. Svg is still required and we should pass it in all stories. 

We need a test/have a test to ensure that not passing an svg doesn't error, but at the same time, svg is still a required prop. Idk about anyone else but I think there is a middle ground there. I expect prop violations in our testing suite to test the middle ground. (although we should have a way to supress them as its messy)

Reply:
@dr3 thanks for the input, I wasn't sure why we had a story for not passing an SVG because it's not an actual use case but is a "fail gracefully" scenario. 

I'll remove this story and add a test to support this. 

+1 on suppressing the output of failing propTypes in the test suite

Reply:
Turns out there is already a unit test for not passing an SVG 🙃